modules/installation-aws-config-yaml.adoc

```diff
@@ -202,27 +217,27 @@ ifdef::openshift-origin[]
 pullSecret: '{"auths":{"<local_registry>": {"auth": "<credentials>","email": "you@example.com"}}}' <12>
 endif::openshift-origin[]
 endif::restricted[]
-ifdef::gov[]
+ifdef::gov,secret[]
```

Comment:
The additional trust bundle is only needed for the secret partition not the government partition.
```suggestion
ifdef::secret[]
```

Reply:
Thank you. Done.